src/modules/Common/components/Instances.tsx

```diff
@@ -67,14 +67,24 @@ const Instances: React.FC<InstancesProps> = ({ children, ...props }) => {
             <>
               {maintainers.map((maintainer) => (
                 <img
-                  key={`maintainer_${kebabCase(maintainer.name)}`}
+                  key={`maintainer_${slugify(maintainer.name)}`}
                   src={maintainer.logo}
                   alt={maintainer.name}
                 />
               ))}
             </>
           );
 
+        const languagesList = languages
+          .map((languageCode) => {
+            if (languageCode === '*') {
+              return t('instances:language.various');
```

Comment:
> I would love for us to have a language code `EU` that would translate to `languages of the European Union` ☺️

Reply:
`Intl.DisplayNames.prototype.of()` function for `language` type could use ISO 639-1 or ISO 639-2 but neither is suitable. 
- the [ISO 639-1 iso code](https://fr.wikipedia.org/wiki/Liste_des_codes_ISO_639-1) `eu` is for Basque language
- in [ISO 639-2](https://fr.wikipedia.org/wiki/Liste_des_codes_ISO_639-2) no code corresponds to European languages 

I can add a custom value `*eu`, what do you think @MattiSG ?

Reply:
Implemented with https://github.com/ambanum/OpenTermsArchive.org/pull/173/commits/8c90de6ed6f2faa9dea47a175d80a01c0925a3f4